spec/services/step_nav_publisher_spec.rb

```diff
@@ -29,6 +31,40 @@
         expect(Services.publishing_api).to have_received(:put_content).with(content_id, payload)
       end
     end
+
+    context "marks draft updated" do
+      let(:step_nav) { create(:step_by_step_page_with_steps) }
+
+      it "does not mark submitted for 2i review as draft" do
+        step_nav.update_attributes(:status => "submitted_for_2i", :review_requester_id => review_requester_user.uid, :reviewer_id => reviewer_user.uid)
+        StepNavPublisher.update(step_nav)
+        expect(step_nav.status).to eq "submitted_for_2i"
+      end
+
+      it "does not mark in 2i review as draft" do
+        step_nav.update_attributes(:status => "in_review", :review_requester_id => review_requester_user.uid, :reviewer_id => reviewer_user.uid)
+        StepNavPublisher.update(step_nav)
+        expect(step_nav.status).to eq "in_review"
+      end
+
+      it "marks 2i approved as draft" do
```

Comment:
Again, think this would be clearer as `it "should revert to draft if a step by step in 'approved_2i' state is edited"`.